### PR TITLE
Init 17 back end task limit start end

### PR DESCRIPTION
**Pull request by IbrahimQumseya**

### CODE REVIEW

File: src/auth/dto/signup-credentials.dto.ts [hunk 1]
```diff
@@ -22,17 +49,11 @@ export class AuthSignUpCredentialsDto {
...
-  @MinLength(2)
-  @MaxLength(20)
+
   firstName: string;
 
-  @IsString()
-  @MinLength(2)
-  @MaxLength(20)
+
   lastName: string;
```
Comment (sergiutoma) [inline src/auth/dto/signup-credentials.dto.ts:58]: Is it a reason why I removed the checks from email, firstname and lastname? 🤔

**Review by sergiutoma — APPROVED** [review]
It looks good!👍 👏

File: src/auth/dto/signup-credentials.dto.ts (hunk 1 above)
Comment (IbrahimQumseya) [reply src/auth/dto/signup-credentials.dto.ts:58]: yeah its the reason so antonia can use it

Comment (sergiutoma) [reply src/auth/dto/signup-credentials.dto.ts:58]: I think we should at least have @IsString() 😉

**Pull request merged**